oak-search/src/main/java/org/apache/jackrabbit/oak/plugins/index/search/FulltextIndexConstants.java

```diff
@@ -401,4 +401,11 @@ public static IndexingMode from(String indexingMode) {
      * cost). The value is: nodes, the path. For properties, the path of the node, then '@' property.
      */
     String USE_IF_EXISTS = "useIfExists";
+
+    /*
+    When set to true for any particular property in an index, queries would support leading wild cards in searches.
+    This is supported in lucene implementation by default, this configuration is primarily meant for elastic index
+    and would be NOOP in lucene index definitions.
+     */
```

Comment:
to make it consistent with the other comments
```suggestion
    /**
     * When set to true for any particular property in an index, queries would support leading wild cards in searches.
     * This is supported in lucene implementation by default, this configuration is primarily meant for elastic index
     * and would be NOOP in lucene index definitions.
     */
```

Reply:
done